dynawo/sources/Common/DYNTrace.cpp

```diff
@@ -227,9 +231,9 @@ void Trace::resetPersistantCustomAppenders_() {
     return;
   }
   TraceSinks& traceSink = sinks_.at(currentId);
-  for (vector< boost::shared_ptr<FileSink> >::const_iterator itSinks = traceSink.persistantSinks.begin();
+  for (std::unordered_map<std::string, boost::shared_ptr<FileSink> >::const_iterator itSinks = traceSink.persistantSinks.begin();
```

Comment:
for (auto itSinks... would really improve the readability here (same for all loops modified)